functions/divide.h

```diff
@@ -0,0 +1,56 @@
+#ifndef FUNCTIONS_DIVIDE_H
+#define FUNCTIONS_DIVIDE_H
+
+// Usage: Divide<F, V>
+// Divide F by V
+// If V = 0, returns 0
+// F, V: FUNCTION, 
+// return value: FUNCTION
+
+class BladeBase;
+
+template<class F, class V>
+class DivideBase {
+public:
+  void run(BladeBase* blade) {
+    f_.run(blade);
+    v_.run(blade);
+  }
+  int getInteger(int led) {
+    if (v_.getInteger(led) == 0) {
+      return 0;
+    } else {
+      return (f_.getInteger(led) / v_.getInteger(led));
```

Comment:
remove extra parenthesis
(same for the SVF function below.)

Reply:
updated